scripts/west_commands/blobs.py

```diff
@@ -63,9 +63,11 @@ def do_add_parser(self, parser_adder):
 
         # Remember to update west-completion.bash if you add or remove
         # flags
-        parser.add_argument('subcmd', nargs=1, choices=['list', 'fetch'],
+        parser.add_argument('subcmd', nargs=1,
+                            choices=['list', 'fetch','clean'],
                             help='''Select the sub-command to execute.
-                            Currently only list and fetch are supported.''')
+                            Currently only list, fetch and clean are
+                            supported.''')
```

Comment:
Why the `only` ?
gives an impression this is due to some kind of limitation, how about:
```suggestion
                            Sub-commands available:
                            - list: list all modules providing binary blobs
                            - fetch: fetch binary blobs
                            - clean: remove downloaded binary blobs
                            for detailed help, run: 'west help blobs <sub-command>'.''')
```

Reply:
Sounds good!

Reply:
Modified it slightly.